index.html

```diff
@@ -1407,7 +1407,7 @@ <h5>
             <p its-locale-filter-list="zh-hant" lang="zh-hant">句號<span class="uname" translate="no">U+3002 IDEOGRAPHIC FULL STOP </span>[。]表示語句結束，逗號<span class="uname" translate="no">U+FF0C FULLWIDTH COMMA </span>[，]表示語氣停頓，頓號<span class="uname" translate="no">U+3001 IDEOGRAPHIC COMMA</span> [、]使用於並列連用、表示次序的字詞之間。</p>
 
             <div class="note" id="n021">
-              <p its-locale-filter-list="en" lang="en">In many college textbooks, science and technology literature, and grammar books of Western languages for example, most of which are in horizontal writing mode, where Western language are heavily used. In these cases, <span class="uname" translate="no">U+FF0E FULLWIDTH FULL STOP</span> [．] can be used as periods, while <span class="uname" translate="no">U+002C COMMA</span> [,] or <span class="uname" translate="no">U+FF0C FULLWIDTH COMMA</span> [，] can be used as commas or secondary commas.</p>
+              <p its-locale-filter-list="en" lang="en" class="checkme">In many college textbooks, science and technology literature, and grammar books in European text for example, most of which are in horizontal writing mode, where European text are heavily used. In these cases, <span class="uname" translate="no">U+FF0E FULLWIDTH FULL STOP</span> [．] can be used as periods, while <span class="uname" translate="no">U+002C COMMA</span> [,] or <span class="uname" translate="no">U+FF0C FULLWIDTH COMMA</span> [，] can be used as commas or secondary commas.</p>
```

Comment:
The “European” here is aimed to translate the Chinese “西(文)”, of which the Japanese counterpart is “欧(文)”. The adoptions of 欧文 can also be found in Taiwan’s typographical publications.

The “欧” above is literally the “European”.

Traditionally, “西(文)” would be translated into “Western”. Yet, the final agreement from CLReq editors’ meeting was to replace the “Western” with the “European”, in order to get away from some potential controversies about the border between the East and the West (or something geopolitical).

Personally, the script set which the term “non-Chinese” refers appears to be much larger than that the “Western” or “European” refers. Considering the Chinese term “西文” usually refers to the scripts like Latin (and Latin extended), Cyrillic, Greek, etc., I also prefer “European” more than “non-Chinese”.

Reply:
Thanks for the explanation of the background - of which i was aware.  But i suggest 'non-Chinese' in the interest of accuracy, rather than 'political' considerations.  It seems like the behaviour of the features under discussion may not be limited to supporting languages of Europe(?)

Reply:
Thank you, Richard. Will discuss this issue with other editors.

Reply:
I discussed this with @r12a offline. He suggested that we use "Latin, Greek, and Cyrillic" directly, or use a phrase like "simple alphabetic text" (with a definition in the document).

Reply:
At yesterday's meeting, we decided to change "European" back to "Western".